Source/reactions/Castro_react.cpp

```diff
@@ -111,7 +117,12 @@ Castro::react_state(MultiFab& s, MultiFab& r, Real time, Real dt, const int stra
             burn_state.T = U(i,j,k,UTEMP);
 
             burn_state.T_fixed = -1.e30_rt;
-
+	    
+#ifdef NSE_NET
+	    burn_state.mu_p = ChemPot(i,j,k,CMUP);
+	    burn_state.mu_n = ChemPot(i,j,k,CMUN);
+#endif
+	    
```

Comment:
I think that after the call to the burner, we need to store the updated chemical potentials back into `ChemPot`